tools/accuracy_checker/accuracy_checker/representation/segmentation_representation.py

```diff
@@ -163,7 +163,7 @@ def areas(self):
         masks = self.mask
         areas = []
         for mask in masks:
-            areas.append(pycocotools.mask.area(mask))
+            areas.append(maskUtils.area(mask))
```

Comment:
Hmm, does this actually fix the problem? I think `maskUtils` is still undefined in this scope.

Reply:
On my view, the right way to handle the problem is storing maskUtils as an attribute of the class. My solution: https://github.com/opencv/open_model_zoo/pull/338

Reply:
What is the downside of importing maskUtils at a global scope?
Does it have a large memory footprint or a long startup time?

Reply:
This module installation has many troubles on some operation systems (e. g. Windows). Accuracy Checker flow wider than evaluation on coco only and in majority cases we do not use maskUtils at all, so I think it is wrong to force a user to handle this heavy dependency if he does not use pycocotools.